Makefile

```diff
@@ -1,21 +1,59 @@
-default: build run 
+SHELL := /bin/bash
+DATABASE_VOLUME := food
 
+PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
+CONTAINER_ID = $(shell docker ps -q --filter "name=tummy" --filter "status=running")
+
+default: build run
```

Comment:
Should we display the help message by default?

Reply:
We should actually show it if the someone runs a target which does not exist. need to implement that